kuma/wiki/templates/wiki/document.html

```diff
@@ -172,7 +172,8 @@
             <a class="button from-search-next only-icon disabled" title="{{ _('Next Search Result') }}" data-empty-title="{{ _('No Previous Search Result') }}"><i aria-hidden="true" class="icon-chevron-right"></i></a>
           </span>
 
-        <h1>{{ document.title }}</h1>
+        {% set safe_title = document.title|forceescape %}
+        <h1>{{ safe_title|addbreaks|safe }}</h1>
```

Comment:
Okay, I think I found out why ...

When `jingo` [registers a filter](https://github.com/jbalogh/jingo/blob/ad909e4830bff6acb1c3f5ca983b6e9c7bf45454/jingo/__init__.py#L119-L137), it [initializes its own jinja2 execution environment](https://github.com/jbalogh/jingo/blob/ad909e4830bff6acb1c3f5ca983b6e9c7bf45454/jingo/__init__.py#L46-L74), with [`autoescape` set to `True`](https://github.com/jbalogh/jingo/blob/ad909e4830bff6acb1c3f5ca983b6e9c7bf45454/jingo/__init__.py#L56). The only way to override the `autoescape` setting of filters would be a global `settings.py` value, which we don't want to do. :cry:

I'll assign this to me and try to think of something we can do here. Or maybe I'll ask dev-webdev.